app/scripts/metamask-controller.js

```diff
@@ -5789,8 +5788,7 @@ export default class MetamaskController extends EventEmitter {
 
           return undefined;
         },
-        getChainPermissionsFeatureFlag: () =>
-          Boolean(process.env.CHAIN_PERMISSIONS),
+        getChainPermissionsFeatureFlag: () => true,
```

Comment:
Think we can just get rid of this entirely?

Reply:
We need it for mocks, can we do that?

Reply:
Sorry I'm not understanding. If this always return true now, then we can just remove all references to `getChainPermissionsFeatureFlag` including the usage in the switch/add ethereum chain handlers and respective tests? Or am I missing something?

Reply:
Ah yes! My bad. We can remove it then